framework/source/class/qx/event/handler/DragDrop.js

```diff
@@ -615,13 +615,16 @@ qx.Class.define("qx.event.handler.DragDrop",
 
       // find current hovered droppable
       var el = e.getTarget();
+      if (this.__startConfig.target === el) {
+        el = document.elementFromPoint(e.getDocumentLeft(), e.getDocumentTop());
+      }
```

Comment:
That could happen in iframes a assume. Good finding I'll change my PR accordingly.